src/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../tsconfig.json",
```

Comment:
Can you explain why we need another tsconfig file?

Reply:
The idea here is to make an incremental migration in the codebase. We could add the `strict` flag to the tsconfig in the root, but this would compel us to make the migration in one go, which would be a lot of work to tackle at once and would generate a hard to review PR. With this approach, we can make easier PRs and break the work required in smaller chunks. WDYT?

Reply:
Makes total sense!